tests/v3/covid-19/api_nyt.spec.js

```diff
@@ -80,6 +80,15 @@ describe('TESTING /v3/covid-19/nyt/counties', () => {
 			});
 	});
 
+	it('/v3/covid-19/nyt/counties lastdays param', (done) => {
+		chai.request(app)
+			.get('/v3/covid-19/nyt/counties?lastdays=1')
+			.end((err, res) => {
+				testBasicProperties(err, res, 200, 'array');
+				done();
+			});
```

Comment:
maybe also check if the length of the returned response is less than that of a normal counties length (probably in the 1000s), and also if it is greater than 1 just to make sure the param is working